packages/arb-validator-core/ethbridge/arbRollupWatcher.go

```diff
@@ -425,23 +533,32 @@ func (vm *ethRollupWatcher) GetParams(ctx context.Context) (valprotocol.ChainPar
 		return valprotocol.ChainParams{}, err
 	}
 	return valprotocol.ChainParams{
-		StakeRequirement:        stakeRequired,
-		GracePeriod:             common.TimeTicks{rawParams.GracePeriodTicks},
+		StakeRequirement: stakeRequired,
+		GracePeriod: common.TimeTicks{
+			Val: rawParams.GracePeriodTicks,
+		},
 		MaxExecutionSteps:       rawParams.MaxExecutionSteps,
 		MaxBlockBoundsWidth:     rawParams.MaxBlockBoundsWidth,
 		MaxTimestampBoundsWidth: rawParams.MaxTimestampBoundsWidth,
 		ArbGasSpeedLimitPerTick: rawParams.ArbGasSpeedLimitPerTick.Uint64(),
 	}, nil
 }
 
-func (vm *ethRollupWatcher) InboxAddress(ctx context.Context) (common.Address, error) {
+func (vm *ethRollupWatcher) InboxAddress(
+	ctx context.Context,
+) (common.Address, error) {
 	addr, err := vm.ArbRollup.GlobalInbox(nil)
 	return common.NewAddressFromEth(addr), err
 }
 
-func (con *ethRollupWatcher) GetCreationInfo(ctx context.Context) (*common.BlockId, common.Hash, error) {
+func (con *ethRollupWatcher) GetCreationInfo(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [receiver name con should be consistent with previous receiver name vm for ethRollupWatcher](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)